superset-frontend/src/pages/ChartCreation/index.tsx

```diff
@@ -299,22 +327,44 @@ export class ChartCreation extends React.PureComponent<
         mouseEnterDelay={1}
```

Comment:
why don't use `import { newLabel } from 'src/dashboard/components/nativeFilters/DatasetLabel` ?
How about create a separate component for newLabel btw?